### PR TITLE
[7.5] Add `elasticsearch/enrich` metricset (#14243)

### PR DESCRIPTION
**Pull request by ycombinator**
Backports the following commits to 7.5:
 - Add `elasticsearch/enrich` metricset  (#14243)

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -89,6 +89,72 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- Add AWS SQS metricset. {pull}10684[10684] {issue}10053[10053]
```
Comment (cachedout) [inline CHANGELOG.next.asciidoc]: This looks like it contains a bunch of unrelated changelog entries?

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Yup, bad rebase. Fixing...

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Fixed in a7c312d.

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
Travis CI is green. Jenkins CI failures are unrelated to this PR. Merging...

**Pull request merged**